assets/components/atoms/icon/icon-list.twig

```diff
@@ -0,0 +1,10 @@
+<table class="table bg-gray-100">
```

Comment:
Not the sexiest background, white seems better 😅

Reply:
it's debug code, indeed needs to go